### ISSUE TITLE
Priority 1 accessibility issue: Colour filter labels

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6563]
**What**

The colour filter relies on colour alone to indicate meaning to users and have no visible text label. Users with limited colour vision or those using limited-colour or monochrome displays may not be able to perceive this information. The buttons also do not have an accessible name to present their purpose to screen reader users nor to allow voice activation users to reference the components directly.

Upon selection of a colour, the page automatically updates with the corresponding results, but there is nothing to confirm this selection to screen reader users. The buttons do not have a ‘selected’ state. If the ‘Colour’ is the only filter applied, no information appears on the page under ‘Active filters’, unlike for the other filters, and the only way to remove these filters is to clear it from within the ‘Colours’ drop down. If another filter is applied, the text ‘Colour’ appears on the page with an adjacent square styled with the corresponding CSS background colour but this has no text alternative for users who cannot perceive this styling.

The hex code of the selected colour appears after the slider when the colour is changed. However, there is nothing to present its association with the buttons to screen reader users, and it is noted that not all users would understand hex codes in order to make this association. This is also not announced to screen reader users when it changes.

<img width="584" alt="Screenshot 2021-05-27 at 16 51 56" src="https://user-images.githubusercontent.com/7315126/119857901-db510e80-bf0b-11eb-9a18-7cca98493724.png">

<img width="262" alt="Screenshot 2021-05-27 at 16 52 22" src="https://user-images.githubusercontent.com/7315126/119857964-ead05780-bf0b-11eb-95ac-1d0c26de150a.png">

<img width="578" alt="Screenshot 2021-05-27 at 16 52 56" src="https://user-images.githubusercontent.com/7315126/119858044-ffaceb00-bf0b-11eb-9669-1fb80ce5ca61.png">

**Where**

Image search results
